lego/apps/lending/managers.py

```diff
@@ -0,0 +1,44 @@
+from lego.utils.managers import BasisModelManager
+
+
+class LendingInstanceManager(BasisModelManager):
+    def create(self, *args, **kwargs):
+        from lego.apps.lending.notifications import LendingInstanceNotification
+        from lego.apps.users.models import Membership, User
+
+        lending_instance = super().create(*args, **kwargs)
+        abakus_groups = lending_instance.lendable_object.responsible_groups.all()
+        roles = lending_instance.lendable_object.responsible_roles
+        if roles:
+            users_to_be_notified = Membership.objects.filter(
+                abakus_group__in=abakus_groups, role__in=roles
+            ).values_list("user", flat=True)
+        else:
+            users_to_be_notified = Membership.objects.filter(
+                abakus_group__in=abakus_groups
+            ).values_list("user", flat=True)
+        for user_id in users_to_be_notified:
+            user = User.objects.get(pk=user_id)
+            notification = LendingInstanceNotification(
+                lending_instance=lending_instance,
+                user=user,
+            )
+            notification.notify()
+
+        return lending_instance
+
+    # TODO: We probably need another template for accepting instances
```

Comment:
I don't understand what it means, but remember to do it or remove the TODO.

Reply:
Removed it for now.